src/presentation/controller/registration/Routing.ts

```diff
@@ -14,6 +14,7 @@ export const EMAIL_TEMPLATE = RegistrationPageType.email;
 export const GENERAL_PARTNERS_TEMPLATE = RegistrationPageType.generalPartners;
 
 export const NEXT_TEMPLATE = RegistrationPageType.next;
+export const LIMITED_PARTNERS_TEMPLATE = RegistrationPageType.limitedPartners;
```

Comment:
can you move it to line 15
should stay in order
the next page is always the last one for the moment

Reply:
order changed